common/tools/dev-tool/src/commands/run/bundle.ts

```diff
@@ -72,7 +77,7 @@ export default leafCommand(commandInfo, async (options) => {
       const bundle = await rollup.rollup(baseConfig);
 
       await bundle.write({
-        file: "dist/index.js",
+        file: `dist/index.${options["output-cjs-ext"] ? "cjs" : "js"}`,
```

Comment:
Instead of using an option, could we just read the main path from package.json?

```suggestion
        file: info.packageJson.main
```

It may be best to defensively check that this field is set and starts with `dist` just to make sure we don't blow up someone's machine from a nasty config.

Reply:
Great point! I changed to use the main path.  We already have the linter to ensure the main path so I just did a minimal check for non-falsy values.